web/services/subscriptions.go

```diff
@@ -1,70 +1,60 @@
 package services
 
 import (
-	log "github.com/sirupsen/logrus"
+	"gorm.io/gorm"
 
-	consulApi "github.com/hashicorp/consul/api"
-	"github.com/trento-project/trento/internal/consul"
-	"github.com/trento-project/trento/internal/subscription"
+	"github.com/trento-project/trento/web/entities"
+	"github.com/trento-project/trento/web/models"
 )
 
 const (
 	SlesIdentifier string = "SLES_SAP"
-	Premium        string = "Premium"
-	Free           string = "Free"
 )
 
-type SubscriptionData struct {
-	Type            string
-	SubscribedCount int
-}
-
 //go:generate mockery --name=SubscriptionsService --inpackage --filename=subscriptions_mock.go
 type SubscriptionsService interface {
-	GetSubscriptionData() (*SubscriptionData, error)
-	GetHostSubscriptions(host string) (subscription.Subscriptions, error)
+	IsTrentoPremium() (bool, int64, error) // Returns if trento is premium or not and the number of subscribed SLES_SAP machines
```

Comment:
`IsTrentoPremium` sounds to me like a `bool, error` return, maybe this could be `GetPremiumData`? Bear in mind we can return a struct with this shape

```go
type PremiumData struct {
    Premium    bool
    Instances  int64
}
```

What do you think?

Reply:
This was the way actually it was implemented before, and I had big doubts on changing it.
I don't have any issue changing it back (using more or less the same structure you mentioned).
Any other opinion?

Reply:
I agree with @dottorblaster.

A `IsTrentoPremium` would be useful to do something like
```
// pseudocode
if is trento premium
    do this
otherwise 
    do that
```
I see it clearer in this context to have the `GetPremiumData` or the `GetSubscriptionData` you already had.
Maybe we need also a `IsTrentoPremium`, probably not for this feature.